Source/Scene/Model.js

```diff
@@ -1186,8 +1187,16 @@ define([
         var cacheKey = defaultValue(options.cacheKey, getAbsoluteUri(url));
 
         options = clone(options);
-        options.basePath = getBaseUri(url, true);
-        options.cacheKey = cacheKey;
+        if (defined(options.basePath)) {
```

Comment:
This works! Just had to change the condition to use `defined(options.basePath)` without the not.